src/Project.ts

```diff
@@ -198,15 +223,13 @@ export default class Project {
       this.stopAllSounds();
       this.runningTriggers = [];
 
-      for (const spriteName in this.sprites) {
-        const sprite = this.sprites[spriteName]!;
-        sprite.clones = [];
-      }
+      this.filterSprites((sprite) => {
+        if (!sprite.isOriginal) return false;
 
-      for (const sprite of this.spritesAndStage) {
         sprite.effects.clear();
         sprite.audioEffects.clear();
-      }
+        return true;
+      });
```

Comment:
Kinda dull, can we separate clearing the effects to happen after `filterSprites` + does this still affect the stage?

```js
this.filterSprites(sprite => sprite.isOriginal);
this.forEachTarget(target => {
  target.effects.clear();
  target.audioEffects.clear();
});
```